lametro/views.py

```diff
@@ -341,7 +341,50 @@ class LABoardMembersView(CouncilMembersView):
 
     def map(self):
 
-        return {}
+        maps = {'map_geojson': {'type': 'FeatureCollection',
```

Comment:
Can you call this `map_geojson_districts` for consistency?

Reply:
there's a bunch of template code that expects there to be an object called `map_geojson` I'm happy to make this change, but it is a bigger change.

Reply:
Hm, I can live without it, if it's going to take that much work.

Reply:
it's not too much work, just wanted to flag that it's not a one-line change.

Reply:
Got it. Holler at me when it's ready!